world.js

```diff
@@ -32,6 +32,61 @@ const localMatrix = new THREE.Matrix4();
 const localMatrix2 = new THREE.Matrix4();
 const localRaycaster = new THREE.Raycaster();
 
+let pendingAnalyticsData = {};
+setInterval(() => {
```

Comment:
Can we abstract this into an `web-monetization.js`? The code looks good but it seems separate from `world.js`.

Reply:
14e3d33